src/content/platform/limits.md

```diff
@@ -26,14 +26,15 @@ order: 2
 
 <TableWrap>
 
-| Feature                  | Free       | Bundled    |
-|--------------------------|------------|------------|
-| [Reads/second](#kv)      | unlimited  | unlimited  |
-| [Writes/second/key](#kv) | 1          | 1          |
-| [Namespaces](#kv)        | 100        | 100        |
-| [Keys/namespace](#kv)    | unlimited  | unlimited  |
-| [Key size](#kv)          | 512 bytes  | 512 bytes  |
-| [Key metadata](#kv)      | 1024 bytes | 1024 bytes |
+| Feature                               | Free       | Bundled    |
+|---------------------------------------|------------|------------|
+| [Reads/second](#kv)                   | unlimited  | unlimited  |
+| [Writes/second](#kv) (different keys) | unlimited  | unlimited  |
```

Comment:
Any reason "(different keys)" isn't included in the URL text? As in:

```
[Writes/second (different keys)](#kv)
```

Everything else in the left column in all these tables is fully linkified

Reply:
Good point, changed.